userland/tools/elf2tab/src/main.rs

```diff
@@ -0,0 +1,396 @@
+extern crate chrono;
+extern crate elf;
+extern crate getopts;
```

Comment:
I would like to get rid off the `getopts crate` and replace it with `docopt` for two reasons:

1. Much nicer help menu and explanation for options etc
2. De-serialize arguments with `serde` 

But it provides some functionality that I don't understand?